nuxeo-services/login/nuxeo-platform-login-jwt/src/main/java/org/nuxeo/ecm/jwt/JWTServiceConfigurationDescriptor.java

```diff
@@ -29,20 +29,20 @@
 @XObject(value = "configuration")
 public class JWTServiceConfigurationDescriptor {
 
-    public static final int DEFAULT_MAX_TTL = 60 * 60; // 1h
+    public static final int DEFAULT_DEFAULT_TTL = 60 * 60; // 1h
 
     @XNode("secret")
     public String secret;
 
-    @XNode("maxTTL")
-    public Integer maxTTL;
+    @XNode("defaultTTL")
+    public Integer defaultTTL;
 
     public String getSecret() {
         return secret;
     }
 
-    public int getMaxTTL() {
-        return maxTTL == null ? DEFAULT_MAX_TTL : maxTTL.intValue();
+    public int getDefaultTTL() {
+        return defaultTTL == null ? DEFAULT_DEFAULT_TTL : defaultTTL.intValue();
```

Comment:
`intValue` is unnecessary due to autoboxing

Reply:
I have IDE warnings about auto-unboxing, which is error-prone, so I prefer to be explicit.